modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java

```diff
@@ -122,17 +122,17 @@ private static String _getInvokerFileNameRegex(
 		).put(
 			"FDSDateFilter", "/data-set-manager/date-filters"
 		).put(
-			"FDSDynamicFilter", "/data-set-manager/dynamic-filters"
+			"FDSDynamicFilter", "/data-set-manager/selection-filters"
 		).put(
 			"FDSEntry", "/data-set-manager/entries"
 		).put(
-			"FDSField", "/data-set-manager/fields"
+			"FDSField", "/data-set-manager/table-sections"
 		).put(
 			"FDSListSection", "/data-set-manager/list-sections"
 		).put(
 			"FDSSort", "/data-set-manager/sorts"
 		).put(
-			"FDSView", "/data-set-manager/views"
+			"FDSView", "/data-set-manager/data-sets"
```

Comment:
These are not persisted, so we can change them without FF. We just need to make sure both old and new UI uses  new endpoints.